changelog/README.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 24th January 2024
+* Extended [Rule](../operations/rules.md#rule) with `ServiceId`.

```

Comment:
nitpick: Consider mentioning the relevant endpoint in the changelog rather than just the entity without context:

```suggestion
* Extended [Get all rules](../operations/rules.md#rule) response with `ServiceId`.
```

Reply:
Reason why I would mention entity is that it can be used by multiple endpoints, in that case you would have to check all the docs if it is returned by one or more and mention every time all the endpoints.

Reply:
Yeah, that was my suspicion. So how about adding at least a noun?
```suggestion
* Extended [Rule](../operations/rules.md#rule) object with `ServiceId`.
```

Reply:
> Reason why I would mention entity is that it can be used by multiple endpoints, in that case you would have to check all the docs if it is returned by one or more and mention every time all the endpoints.

Yes I agree @VBuben, why not do that? For example, "Extended Rule with ServiceId, this change affects the following operations: ..."

Reply:
@MikeAdamsMews what about now? Even it is used in one place, I would keep it as is to be consistent in the future when there are multiple endpoints. What do you think?